sale_report_delivered/views/sale_report_delivered_views.xml

```diff
@@ -51,6 +51,7 @@
                     domain="[('state','not in',('draft', 'cancel', 'sent'))]"
                 />
                 <separator />
+                <field name="user_from_partner_id" />
```

Comment:
Please, can you put the new field after de user_id?
Users are used to having that field first.

Reply:
Done!